Automattic/wp-calypso
# HelpCenter: add to wp-admin bar

## Proposed Changes

![Screen Capture on 2022-06-23 at 00-54-04](https://user-images.githubusercontent.com/33258733/175330126-c65a113d-c8e1-441d-92a5-ceca77d761f6.gif)

This is the next phase of the HelpCenter release. This injects the HelpCenter component into the wp-toolbar. I followed the notification-ui example for inspiration.

In order to get the site-id I had to update our method for grabbing it. Before we relied on `window._currentSiteId`. Since this was not set in the `/wp-admin` page I used the `wp_localize_script` to set it as a variable and set it in the claypso provider component.

Closes #63700

**Things to address**
- [ ] Verify with smarter people if this method is safe and ok.
- [ ] Add a bit more logic to `apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php` to prevent this from loading in places it shouldn't
- [ ] See if `wp_add_inline_script` is better option.
- [ ] Should this be on mobile?

## Testing Instructions

1. Pull branch
2. `cd apps/editing-toolkit/ && yarn dev --sync`
3. Go to `/wp-admin` for simple site
4. Verify that the help center is showing properly
5. All features of the help center should be working including support.
6. [Download ETK](https://teamcity.a8c.com/repository/download/calypso_WPComPlugins_EditorToolKit/8115973:id/editing-toolkit.zip) and install on Atomic to test the same thing
